charts/filebeat/ci/test-values.yaml

```diff
@@ -120,12 +128,113 @@ filebeatConfig:
+  standardConfig:
+    winlogbeat.yml: |-
+      winlogbeat.event_logs:
+        - name: Application
+          ignore_older: 72h
+        - name: Security
+        - name: System
+
+      fields:
+        logzio_codec: ${LOGZIO_CODEC}
+        token: ${LOGZIO_LOGS_SHIPPING_TOKEN}
+        type: ${LOGZIO_TYPE}
+      fields_under_root: ${FIELDS_UNDER_ROOT}
+
+      processors:
+        - rename:
+            fields:
+            - from: "agent"
+              to: "winlogbeat_agent"
+            ignore_missing: true
+        - rename:
+            fields:
+            - from: "log.file.path"
+              to: "source"
+            ignore_missing: true
+        - rename:
+            fields:
+            - from: "log"
+              to: "log_information"
+            ignore_missing: true
+
+      output.logstash:
+        hosts: ["${LOGZIO_LOGS_LISTENER_HOST}:5015"]
+        ssl:
+          certificate_authorities: ['C:\cert.crt']
+
+filebeatWindowsConfig:
+  standardConfig:
+    filebeat.yml: |-
+      filebeat.inputs:
+      - type: container
+        enabled: true
+        paths:
+          - c:\var\lib\docker\containers\*\*.log
+        fields:
+          logzio_codec: ${LOGZIO_CODEC}
+          token: ${LOGZIO_LOGS_SHIPPING_TOKEN}
+          type: ${LOGZIO_TYPE}
+        symlinks: true
+        fields_under_root: ${FIELDS_UNDER_ROOT}
+        encoding: utf-8
+        ignore_older: ${IGNORE_OLDER}
+      filebeat.config.modules:
+        path: ${path.config}/modules.d/*.yml
+        reload.enabled: false
+      output.logstash:
+        hosts: ["${LOGZIO_LOGS_LISTENER_HOST}:5015"]
+        ssl.certificate_authorities: ["C:\\cert.crt"]
+      processors:
+      - rename:
+          fields:
+          - from: "agent"
+            to: "filebeat_agent"
+          ignore_missing: true
+      - rename:
+          fields:
+          - from: "log.file.path"
+            to: "source"
+      - add_cloud_metadata: ~
+      loggin.json: true
+  autodiscoverConfig:
+    filebeat.yml: |-
+      filebeat.autodiscover:
+        providers:
+          - type: kubernetes
+            node: ${NODE_NAME}
+            hints.enabled: true
+            hints.default_config:
+              type: container
+              paths:
+                - c:\var\lib\docker\containers\${data.kubernetes.container.id}\*.log
+            include_annotations: '*'
+
+      processors:
+        - add_cloud_metadata: ~
+      fields:
+        logzio_codec: ${LOGZIO_CODEC}
+        token: ${LOGZIO_LOGS_SHIPPING_TOKEN}
+        cluster: ${CLUSTER_NAME}
+        type: ${LOGZIO_TYPE}
+      fields_under_root: ${FIELDS_UNDER_ROOT}
+      ignore_older: ${IGNORE_OLDER}
+      output:
+        logstash:
+          hosts: ["${LOGZIO_LOGS_LISTENER_HOST}:5015"]
+          ssl:
+            certificate_authorities: ['C:\\cert.crt']
+  autoCustomConfig: {}
+
 serviceAccount:
   create: true
   name: filebeat
 
 terminationGracePeriod: 30
 hostNetwork: true
+windowsHostNetwork: false
```

Comment:
What is this parameter and why is it false?

Reply:
It defines if the application running in the pod can see the network interfaces of the host machine, this is option is not yet support on windows.